src/ngraph/op/convolution.cpp

```diff
@@ -584,13 +584,27 @@ Shape op::util::infer_convolution_output_shape(const Node* node,
         << "Output channel count for filters is zero (filters shape: " << filters_shape << ", "
         << "output channels on axis " << output_channel_axis_filters << ").";
 
-    NODE_VALIDATION_ASSERT(node, filters_shape[input_channel_axis_filters] == input_channel_count)
-        << "Input channel count for filters (" << filters_shape[input_channel_axis_filters] << ") "
-        << "does not match the number of channels in the data batch (" << input_channel_count
-        << ") "
-        << "(filter input shape: " << filters_shape << ", filter input channels on axis "
-        << input_channel_axis_filters << "; data batch shape: " << data_batch_shape
-        << ", data batch channels on axis " << batch_axis_data << ").";
+    if (node->description() == "GroupConvolution" || node->description() == "GroupConvolutionBias")
+    {
+        NODE_VALIDATION_ASSERT(node, filters_shape.at(0) == input_channel_count)
+            << "Input channel count for filters (" << filters_shape.at(0) << ") "
+            << "does not match the number of channels in the data batch (" << input_channel_count
+            << ") "
+            << "(filter input shape: " << filters_shape
+            << "; data batch shape: " << data_batch_shape << ").";
+    }
+    else
+    {
+        NODE_VALIDATION_ASSERT(node,
+                               filters_shape[input_channel_axis_filters] == input_channel_count)
+            << "Input channel count for filters (" << filters_shape[input_channel_axis_filters]
+            << ") "
+            << "does not match the number of channels in the data batch (" << input_channel_count
+            << ") "
+            << "(filter input shape: " << filters_shape << ", filter input channels on axis "
+            << input_channel_axis_filters << "; data batch shape: " << data_batch_shape
+            << ", data batch channels on axis " << batch_axis_data << ").";
+    }
 
     //
     // Make sure window movement strides, window dilation strides, and data dilation strides
```

Comment:
Please make sure that there are no mode changes on these files.

Reply:
yes, will do. this was actually a temp commit.